static/templates/settings/deactivate_realm_modal.hbs

```diff
@@ -1,17 +1,2 @@
-<div id="deactivate-realm-modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivate_realm_modal_label" aria-hidden="true">
-    <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="deactivate_realm_modal_label">
-            {{t "Deactivate organization" }} <span class="realm_name"></span>
```

Comment:
Did this `realm_name` spam display the realm name before this change?  

I'm not sure we need it here (we definitely want it in the modal somewhere), but we shouldn't be accidentally changing it in this commit.

Reply:
No, the `realm_name` span didn't actually contain the realm name. 
It's just a empty span added to the modal heading. (Not sure on why was it exactly added!)

Reply:
OK, this would have been a key detail to explain in the commit message.

Can you start a #design thread with that screenshot about where we should put the identity of the organization to deactivate?  I do think we probably want to display it somewhere in that modal.